### PR TITLE
Fixed bug in buggy_function.py + is_prime function

### PR DESCRIPTION
**Pull request by raquel-ruiz**
Missing: improve README and add docstrings

### CODE REVIEW

**Comment by ghurone** [comment]
i'm going to review!

File: participants/Ruiz_Valença/pr_tutorial/buggy_function.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     if math.floor(decimals) != decimals:
         raise OSError('decimals should be an integer!')
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees*24/360
```
Comment (ghurone) [inline participants/Ruiz_Valença/pr_tutorial/buggy_function.py:23]: Nice, you found the bug!

**Review by ghurone — APPROVED** [review]
the code is fine!

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**